Documentation/tls/tls-identity.md

```diff
@@ -0,0 +1,63 @@
+# Enabling custom Tectonic Identity TLS certificates
+
+Use Tectonic's [Identity TLS module][identity-module] to enable user provided Identity certificates.
+
+The module does not contain any logic, but just passes user provided certificates from its input directly to its output. This is to prevent changing existing references to the `tls/identity/self-signed` module, hence all `tls/identity/*` modules share the same outputs.
+
+## Usage
+
+Comment out the existing self-signed identity TLS in your platform, i.e. `platforms/aws/tectonic.tf`:
+
+```
+/*
+module "identity_certs" {
+  source = "../../modules/tls/identity/self-signed"
+
+  ca_cert_pem = "${module.kube_certs.ca_cert_pem}"
+  ca_key_alg  = "${module.kube_certs.ca_key_alg}"
+  ca_key_pem  = "${module.kube_certs.ca_key_pem}"
+}
+*/
+```
+
+Configure the user provided certificate paths in your platform, i.e. `platforms/aws/tectonic.tf`:
```

Comment:
These certs will be used to secure communication between the tectonic console and the tectonic identity service grpc endpoint. This is done to ensure that tokens that are generated for cluster access are secured all the time. @ericchiang maybe for a little more color.

Reply:
Tectonic identity provides a gRPC endpoint to do things like revoke tokens. This cert is used for console to communicate with that API.